src/components/PostDetails.tsx

```diff
@@ -2,91 +2,44 @@ import React, { useEffect, useState } from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
 
-import * as commentsApi from '../api/comments';
-
 import { Post } from '../types/Post';
-import { Comment, CommentData } from '../types/Comment';
+import { CommentData } from '../types/Comment';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
+import {
+  addComment,
+  delateComment,
```

Comment:
There is a typo in the action name `delateComment`. It should be `deleteComment` to match the intended functionality and avoid errors.